src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,125 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double INITIAL_LOAD_FACTOR = 0.75;
+    private Entry<K, V>[] table = new Entry[INITIAL_CAPACITY];
```

Comment:
The diamond operator is missing in the `new Entry[INITIAL_CAPACITY]` expression. It should be `new Entry<>[INITIAL_CAPACITY]`.

Reply:
It's impossible, it's array!!!